lucille-core/src/main/java/com/kmwllc/lucille/indexer/ElasticsearchIndexer.java

```diff
@@ -144,6 +174,22 @@ private void addChildren(Document doc, Map<String, Object> indexerDoc) {
     }
   }
 
+  private static VersionType versionTypeFromString(String versionType) {
```

Comment:
VersionType is an enum so we shouldn't have to write this method
can do VersionType.valueOf(str) but that would require the str matches the enum value exactly, so it could be "Internal", "External", "ExternalGte", or "Force"
Do we need to allow the user to enter "internal" "external" "external_get" or "force" ?
If so, we could support it like this:
VersionType.valueOf(CaseUtils.toCamelCase("external_gte", true, '_'))

see https://commons.apache.org/proper/commons-text/apidocs/org/apache/commons/text/CaseUtils.html

Reply:
abi's OpenSearch PR is using VersionType.valueOf without the CaseUtils part
unless there's a known reason why we need to support the version type names in snake case, let's just do VersionType.valueOf() and update any configs to use "Internal", "External", "ExternalGte", or "Force"
See https://github.com/kmwtechnology/lucille/blob/15e83f3375a48b3e9a2eb2bd4c2e41bc6ba95b95/lucille-core/src/main/java/com/kmwllc/lucille/indexer/OpenSearchIndexer.java#L51